release.sh

```diff
@@ -145,21 +153,43 @@ if [ $dodist -eq 0 ]; then
     # We haven't tagged yet, so tell the dist script what version
     # it's building
     DIST_VERSION="$tag" npm run dist
+
+    `dirname $0`/scripts/changelog_head.py > latest_changes.md
```

Comment:
Oops, added changelog_head. And yeah, this should work since it's taking the path the script itself is executed with (which will be node_modules/matrix-js-sdk/ because of https://github.com/vector-im/riot-web/blob/master/release.sh#L12) and executing changelog_head relative to that